internal/featuresupport/features_misc.go

```diff
@@ -287,16 +287,14 @@ func (f *skipMcoReboot) getIncompatibleFeatures(openshiftVersion string) *[]mode
 }
 
 func (f *skipMcoReboot) getIncompatibleArchitectures(openshiftVersion *string) *[]models.ArchitectureSupportLevelID {
-	return &[]models.ArchitectureSupportLevelID{
-		models.ArchitectureSupportLevelIDS390XARCHITECTURE,
-	}
+	return nil
```

Comment:
Is it anticipated that there will be further incompatible architectures in the future?
Why do we need to keep this function at all?

Reply:
Based on the supported architecture I can't give an answer here because I don't know what's coming next nor I can't say if there will be or not be a supporting limitation. In addition we have a limitation to a version. 
I would say we should keep this function and if support is ending for 4.15 we can re-asses this function (I believe there will be more changes needed).